gist/src/main/java/build/gist/data/listeners/Queue.kt

```diff
@@ -54,6 +54,10 @@ class Queue: GistListener {
             .create(GistQueueService::class.java)
     }
 
+    internal fun checkLocalStore() {
```

Comment:
From the function name, I am not understanding what this code's job is. 

Checking local store, for what? 

I believe if this function name was more descriptive, it would improve the code where the function is called. 

Example, in this PR, this code exists:

```kt
    fun setCurrentRoute(route: String) {
        currentRoute = route
        currentRoute = route
        gistQueue.checkLocalStore()
    }
```

From this code snippet, I ask myself, "Why are we checking local store after setting a route? What is check local store doing that is needed in this function after setting a route?"

Reply:
Agreed, renamed method to `fetchUserMessagesFromLocalStore`